content/2021-losangeles/submit.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Submit Talk
+---
+
+## Submit a talk
+
+The PromCon Call for Papers (CfP) is open for talk submissions until 23:59 UTC on
+Sunday, July 25, 2021, and we encourage you to submit a talk!
+
+For any questions regarding the CFP process, please email
+[CNCFcolocatedevents@cncf.io](mailto:CNCFcolocatedevents@cncf.io).
+
+
+<a class="btn btn-default btn-lg" href="https://linuxfoundation.smapply.io/prog/promcon_north_america_2021/" target="_blank" role="button">
+  <i class="fa fa-commenting"></i>&nbsp;&nbsp;Submit Talk
+</a>
+
+## Dates to remember
+
+- CFP opens: Monday, June 28
```

Comment:
Title case the "opens" -> "Opens" like the other bullet points or do it all the other way around?